feature_engine/encoding/decision_tree.py

```diff
@@ -239,6 +281,15 @@ def fit(self, X: pd.DataFrame, y: pd.Series):
         variables_ = self._check_or_select_variables(X)
         _check_contains_na(X, variables_)
 
+        # dictionary of categories and seen values
+        if self.unseen in ["encode", "ignore"]:
```

Comment:
would it not be faster if self.unseen != "raise" ? instead of searching over a list?